decoder/body_candidates.go

```diff
@@ -17,6 +17,18 @@ func (d *PathDecoder) bodySchemaCandidates(body *hclsyntax.Body, schema *schema.
 	candidates := lang.NewCandidates()
 	count := 0
 
+	if schema.Extensions != nil {
+		// check if this schema supports Count attribute
+		if schema.Extensions.Count {
+			countPresent := isAttributePresent(body, "count")
+			// check if Count is already used inside this body, so we don't
+			// suggest a duplicate
+			if !countPresent {
```

Comment:
```suggestion
			if _, ok := body.Attributes["count"]; !ok {
```

We can simplify this by checking directly for the `count` attribute and don't need the `isAttributePresent` helper method. Sorry for the wrong tip to use it.